posthog/middleware.py

```diff
@@ -357,6 +358,11 @@ def __call__(self, request: HttpRequest):
                 return response
             finally:
                 reset_query_tags()
+        else:
```

Comment:
doesn't make sense for this to be in `captureMiddleware` ?

Reply:
If you mean "wouldn't it make sense for this to be in capturemiddleware" I've got good news, which is that it is.

If you mean "It doesn't make sense for this to be in capturemiddleware" I was on the fence on this. The thing is this the proxy should only be using the capture endpoints and they're in a nice list here, so hence me putting it here. Would take suggestions where else to put it!

Reply:
This short-circuits capture, so as long as your middleware goes after it, everything will be _fine_ 😅 .

The other problem is that 'proxy is for capture only' is not 100% correct, as `/decide` is also another endpoint that depends on it.

I'd say make a new middleware, put it after capture & decide shortcircuit both, and if you wish also explicitly add list of endpoints not to redirect on. Maybe you can reuse `ALWAYS_ALLOWED_ENDPOINTS` / update it for this list?

Reply:
Actually, any calls to `local_evaluation` would probably use the same host, wouldn't it? 🤔 

ah, but no, that occurs on servers, so there shouldn't be a proxy, but if people did route to it on their servers as well, we might bork here too.